### DISCUSSION TITLE
is any grafana dashboard compatible with that metrics nginxlog exporter?

### DISCUSSION THREAD

**Discussion by Shaverdoff** [martin-helmich/prometheus-nginxlog-exporter#246] [Q&A]
actually the whole question in the topic

**Comment by martin-helmich** [comment]
One that I know of is [this one](https://grafana.com/grafana/dashboards/6482) (please note that while that dashboard uses the metrics provided by this exporter, I am not affiliated with the dashboard in any other way).

Apart from that, I've supplied some example PromQL queries in [my blog post](https://www.martin-helmich.de/en/blog/monitoring-nginx.html) that you can use to build your own dashboard.

**Comment by brunomarram** [comment]
I was created [this one](https://grafana.com/grafana/dashboards/16155) a week ago too. But it's not documented yet. I pretend to do that as soon as possible.

PS.: it is in portuguese, if you want in english or some other language just translate titles and its done
